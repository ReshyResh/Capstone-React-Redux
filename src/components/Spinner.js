import loading from '../images/loading.gif';

const Spinner = () => <div className="spinner-container"><img src={loading} alt="loading" className="spinner" /></div>;

export default Spinner;