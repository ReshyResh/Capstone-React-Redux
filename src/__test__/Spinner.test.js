import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from '../components/Spinner';

it('Spinner renders correctly', () => {
  const tree = renderer
    .create(<Spinner />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
