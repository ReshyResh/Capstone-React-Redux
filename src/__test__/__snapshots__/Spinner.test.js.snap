// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders correctly 1`] = `
<div
  className="spinner-container"
>
  <img
    alt="loading"
    className="spinner"
    src="loading.gif"
  />
</div>
`;
